seeds/zeroex/polygon/zeroex_polygon_api_fills_deduped_sample.csv

```diff
@@ -1,6 +1,6 @@
-polygon,2023-01-01T00:00:00Z,2023-01-01T07:59:38Z,USDT,0x79375c41d88f839f551457145066096c5c8944bc,,0.4985,,498500,8.796288564730349e+18,0.498831004,0xc2132d05d31c914a87c6611c10748aeb04b58e8f,0x79375c41d88f839f551457145066096c5c8944bc,0x2f2dd99235cb728fc79af575f1325eaa270f0c99,0x00000002517569636b5377617000000000000000,0x1000000000000000000000000000000000000011,0x00b07f278afaa17ea587d7f8aedaa233337e02a309aff6f69c0d9535634c6bbb,0x1d8535a03710b0341ed52ddbdfd57fb0890d4377,0xd1ca1f4dbb645710f5d5a9917aa984a47524f49a,113,BridgeFill,true,1
-polygon,2023-01-01T00:00:00Z,2023-01-01T13:32:27Z,USDT,USDC,USDC-USDT,0.009999,0.009991,9999,9991,0.009996804771,0xc2132d05d31c914a87c6611c10748aeb04b58e8f,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,0xfd896db057f260adce7fd1fd48c6623e023406cd,0x0000000241706553776170000000000000000000,0x1000000000000000000000000000000000000011,0x00b6cdd803a3e1ca2126472a2dd5acb361f0d70a748997b4cbce07edc1affa25,0xfdc8db8dcd2aad69e0943ca6b0986a69a167a9e2,0xfd896db057f260adce7fd1fd48c6623e023406cd,447,BridgeFill,true,1
-polygon,2023-01-01T00:00:00Z,2023-01-01T02:20:45Z,0xc168e40227e4ebd8c1cae80f7a55a4f0e6d66c97,USDC,,,41.3089,1.8752344479089898e+21,4.13089e+07,41.331206806,0xc168e40227e4ebd8c1cae80f7a55a4f0e6d66c97,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,0xc590175e458b83680867afd273527ff58f74c02b,0x000000024466796e000000000000000000000000,0x11ededebf63bef0ea2d2d071bdf88f71543ec6fb,0x00ca8b7d0039030fd1d08480fb5f5c7f7edb47a6e40fee0644b3e05ce01c8fb4,0x4b1fa071e31836f61bf26b10bf4d577345fddcf6,0x1a1ec25dc08e98e5e93f1104b5e5cdd298707d31,147,BridgeFill,true,1
\ No newline at end of file
+blockchain,project,block_date,block_time,taker_symbol,maker_symbol,token_pair,taker_token_amount,maker_token_amount,taker_token_amount_raw,maker_token_amount_raw,volume_usd,taker_token,maker_token,taker,maker,affiliate_address,tx_hash,tx_from,tx_to,evt_index
+polygon,0x API,2023-01-01,2023-01-01 7:02,USDC,DAI,DAI-USDC,98.247675,98.24887947,98247675,98248879472596880000,98.28906326,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,0x8f3cf7ad23cd3cadbd9735aff958023239c6a063,0x0e5789f77945013c3acfc1cee3e1e03d2dd8b8c5,0x0000001942616c616e6365725632000000000000,0x11ededebf63bef0ea2d2d071bdf88f71543ec6fb,0x01615a9f260c70c4e2c2e2e65ea21df56b3bd17034f694cf0525721e1ceaa0d1,0x0e5789f77945013c3acfc1cee3e1e03d2dd8b8c5,0x1a1ec25dc08e98e5e93f1104b5e5cdd298707d31,227
+polygon,0x API,2023-01-01,2023-01-01 0:19,USDC,CRV,CRV-USDC,50,95.0201732,50000000,95020173202461750000,50.00704359,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,0x172370d5cd63279efa6d502dab29171933a610af,0xaf677145f8f03ecd25da06e290c59cb5a64aa163,0x00000012556e6973776170563300000000000000,0x5bd68b4d6f90bcc9f3a9456791c0db5a43df676d,0x02c241e84f97f5389550c94b46ee9187a846401cf887e0d42002bb4e1c458c10,0xaf677145f8f03ecd25da06e290c59cb5a64aa163,0xdef1c0ded9bec7f1a1670819833240f027b25eff,94
```

Comment:
Event 94 in this transaction is exchange of WETH and CRV:
https://polygonscan.com/tx/0x02c241e84f97f5389550c94b46ee9187a846401cf887e0d42002bb4e1c458c10#eventlog

I will update the taker token address.